hollow/src/test/java/com/netflix/hollow/api/codegen/HollowAPIGeneratorTest.java

```diff
@@ -0,0 +1,54 @@
+package com.netflix.hollow.api.codegen;
+
+import com.netflix.hollow.core.write.HollowWriteStateEngine;
+import com.netflix.hollow.core.write.objectmapper.HollowObjectMapper;
+import java.io.File;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class HollowAPIGeneratorTest {
+    private String baseDir = "/tmp/HollowAPIGeneratorTest";
```

Comment:
Might be best to use `System.getProperty("java.io.tmpdir")` instead of hardcoding `"/tmp"`